backend/app/Http/Controllers/QuizLogController.php

```diff
@@ -34,6 +35,14 @@ protected function getUserWithQuizLogs($id) {
     }
 
     protected function validateQuizLog(Request $request) {
+        $checkUnique = QuizLog::where([
+            ['quiz_id', $request->quiz_id],
+            ['user_id', $request->user_id]
+        ])->first();
+        if ($checkUnique) {
```

Comment:
please put new line before condition
```suggestion
        ])->first();
        
        if ($checkUnique) {
```

Reply:
done